src/hatti/views/dataview.cljs

```diff
@@ -190,7 +190,15 @@
           (for [{:keys [component view]} dataviews
                 :let [view-name (name view)]]
             [:div {:class (str "tab-page " view-name "-page")
-                   :style {:display (view->display view)}}
+                   :style {:height
+                           (:table-view-height (-> app-state :table-page))
```

Comment:
I think `(-> app-state :table-page)` can be re-written as `(:table-page app-state)`

Reply:
I will use `(-> app-state :table-page :table-view-height)` as you've suggested bellow.